content/ja/docs/concepts/cluster-administration/proxies.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Kubernetesのプロキシー
+content_type: concept
+weight: 90
+---
+
+<!-- overview -->
+このページではKubernetesと併用されるプロキシーについて説明します。
+
+
+<!-- body -->
+
+## プロキシー
+
+Kubernetesを使用する際に、いくつかのプロキシーを使用する場面があります。
+
+1. [kubectlのプロキシー](/docs/tasks/access-application-cluster/access-cluster/#directly-accessing-the-rest-api):
+
+    - ユーザのデスクトップ上かPod内で稼働する
+    - ローカルホストのアドレスからKubernetes api-serverへのプロキシー
+    - クライアントからプロキシ間ではHTTPを使用します
+    - プロキシーからapi-serverへはHTTPSを使用します
+    - api-serverを配置します
+    - 認証用のヘッダーを追加します
+
+1. [api-serverのプロキシー](/docs/tasks/access-application-cluster/access-cluster/#discovering-builtin-services):
+
+    - api-server内で動作する防壁となります
+    - クラスターの外部からクラスターIPへユーザを接続し、できなければ到達不可能となります
```

Comment:
```suggestion
    - これがなければ到達不可能であるクラスターIPへ、クラスターの外部からのユーザーを接続します
```

1. which otherwise ... reachableがcluster IPsを修飾していると思います
1. userは「ユーザー」でお願いします
1. "a user outside of the cluster" が多分一句で、クラスター外にいるユーザー、ということかと思います

Reply:
>    - connects a user outside of the cluster to cluster IPs which otherwise might not be reachable
たしかに、こちら修正します！